torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -44,7 +47,13 @@ class AQTLayout(TorchAOBaseTensor):
     """
     Base class for the layout tensor for `AffineQuantizedTensor`
     """
-    def get_plain(self) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
+    def get_plain(self) -> Tuple[torch.Tensor, torch.Tensor, Optional[torch.Tensor]]:
+        """ This method will be called when dequantizing the Tensor
+        It should return a tuple 3 things:
+            * The underlying data
```

Comment:
nit: we can add "unpacked" here as well I think

Reply:
also does optional zero point work? I thought None doesn't work in torch.compile

Reply:
yeah compile seems to be working as expected here

Reply:
actually I dont know what you mean by "unpacked"

Reply:
oh it means data is not optimized for storage or computation, e.g. uint8 + quant_min=0, quant_max=15, will be the unpacked representation for uint4, when we pack it we can put 2 elements of uint4 data into one uint8 element.